lib/AccessoryManager.js

```diff
@@ -500,6 +507,200 @@ class AccessoryManager {
     )
   }
 
+  /**
+   * Configure the "Light" service for the accessory
+   */
+  _configureDimLightService () {
+    const lightBulbService = this._getService(this._hapService.Lightbulb)
+    const onCharacteristic = lightBulbService.getCharacteristic(this._hapCharacteristic.On)
+    const brightnessCharacteristic = lightBulbService.getCharacteristic(this._hapCharacteristic.Brightness)
+
+    // Setup handler for when the value for the "On" characteristic is requested / set by HomeKit
+    const onCharacteristicName = 'On'
+    const onCharacteristicZwaveNodeValueCriteria = {
+      commandClass: COMMAND_CLASS_SWITCH_MULTILEVEL,
+      valueIndex: SWITCH_BINARY_INDEX_SWITCH
+    }
+
+     const onCharacteristicValueSet = value => {
+       var setValue = brightnessCharacteristic.value
```

Comment:
Hmm, should this rely be retrieved from the `zwave` node rather than from the characteristic directly? Could there be a condition where the `Brightness` characteristic does not get a value set before being accessed here?